pkg/utils/parse_arf_result.go

```diff
@@ -761,9 +768,9 @@ func toArrayByComma(format string) []string {
 	return strings.Split(format, ",")
 }
 
-//This function will take orginal remediation content, and a list of all values found in the configMap
-//It will processed and substitue the value in remediation content, and return processed Remediation content
-//The return will be Processed-Remdiation Content, Value-Used List, Un-Set List, and err if possible
+// This function will take orginal remediation content, and a list of all values found in the configMap
```

Comment:
LGTM! Just a couple of comments and suggestions left below: 

```suggestion
// This function will take original remediation content, and a list of all values found in the configMap
```